zerver/lib/actions.py

```diff
@@ -17,7 +17,8 @@
 
 from zerver.lib.bugdown import (
     version as bugdown_version,
-    url_embed_preview_enabled_for_realm
+    url_embed_preview_enabled_for_realm,
+    convert as convert_to_bugdown
```

Comment:
I'd call this `bugdown_convert`; we use that name elsewhere when importing it.

Reply:
Done.